Packs/CarbonBlackDefense/Integrations/integration-CarbonBlackDefense.yml

```diff
@@ -1054,7 +1054,7 @@ script:
     - name: ipAddress
       description: Query parameter. Filter on events generated by a device with a
         given external or internal IP address
-    - name: sha256hash
+    - name: sha256Hash
```

Comment:
Why is this needed? This is very dangerous is it can break backward compatibility.